examples/merged/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+    kotlin("jvm") version "1.7.10"
+    id("org.jetbrains.kotlinx.kover") version "0.6.0"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(project(":subproject"))
+    implementation(project(":excluded"))
+    testImplementation(kotlin("test"))
+}
+
+koverMerged {
+    enable()
+
+    filters {
+        classes {
+            includes += "kotlinx.kover.examples.merged.*"
+            excludes += listOf("kotlinx.kover.examples.merged.utils.*", "kotlinx.kover.examples.merged.subproject.utils.*")
+        }
+
+        projects {
+            excludes += "excluded"
```

Comment:
Now it is expected that a plugin for should be applied in all subprojects, otherwise it should be explicitly excluded from reports.

This was specially done, in case the plugin was applied at the root project, users should be notified (with error) that they need to apply the plugin in subprojects by themselves, and not wait for the Kover itself to do it.

Reply:
IMO it's not very convenient when I have a lot of (lets say 100) subprojects and want to apply Kover to half of them — as far as I understand, I need to apply Kover to all 100 and then add 50 of them to excludes?

Anyway, it's out of scope of this PR, so let's discuss this later